### PR TITLE
Bump hammer-cli-katello requirements for hammer-cli-foreman

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -27,7 +27,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.18.0
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 1.0.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 3.0.0
```
Comment (tbrisker) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: The gemspec does not define a version limit, we shouldn't define one in the rpm spec either

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 2]
```diff
@@ -98,6 +98,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Thu Feb 13 2019 Zach Huntington-Meath - 0.21-0.3.pre.master
+- Unify prerelease macro handling
```
Comment (tbrisker) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: this is probably not the changelog you were planning to add :)

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec (hunk 1 above)
Comment (zjhuntin) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: ack

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec (hunk 2 above)
Comment (zjhuntin) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: hah, it was not

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 3]
```diff
@@ -98,6 +97,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Thu Feb 13 2019 Zach Huntington-Meath - 0.21-0.3.pre.master
```
Comment (tbrisker) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Linter doesn't like you using last year in the date ;)

Comment (zjhuntin) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Linter is so picky

**Pull request merged**